### ISSUE TITLE
Lava and water animations

### ISSUE THREAD

**Issue by leorockway** [Wuzzy2/MineClone2-Bugs#212]
Lava should have particles popping out of it every once in a while.
Water should be showing bubbles when the player is underwater.

**Comment by leorockway** [comment]
The underwater bubbles I'm refering to are little particles floating in the water, not like the big ones shown in MineClone in front of the player. It's more similar to the snow or rain particles.

**Comment by Wuzzy2** [comment]
Ugggh, there are actually underwater bubbles, have you not seen *any* of those? Or do you mean that bubbles should spawn everywhere in water, not just at players and mobs?

Lava particles are TODO, of course.

**Comment by leorockway** [comment]
These are the MT bubbles I see: http://i.imgur.com/LxNFs0u.png
They look like the player is breathing and they look cool (and I don't think MC actually has them, but I like them).

These are the MC bubbles I was talking about (they are more like snow, they are floating in the water): http://i.imgur.com/itP5Vwa.png (they are the tiny white dots you see floating and you can move closer to them).

**Comment by leorockway** [comment]
MC actually does have the "breating"-like bubbles from MT, but they show up when the player enters the water (like a splash).

I have noticed that the screen looks like it's covered in water when the player enters water. It's like the whole screen has an image overlay with droplets.

**Comment by Wuzzy2** [comment]
Thanks for your *very* detailed reports, but I tend not to worry about these kind of things for now. Low priority.

**Comment by leorockway** [comment]
Yup, they are definitely not a priority. I'm just reporting what I see. I never really played Minecraft before (I tried it, but barely, as it runs _really_ slow on my computer). I only downloaded the demo so I can report on the differences with Mineclone.

Minetest / MineClone runs very smoothly on my computer (as long as I disable the Shaders).

**Comment by tobyplowy** [comment]
[how items work.zip](https://github.com/Wuzzy2/MineClone2-Bugs/files/1049591/how.items.work.zip)
